app/src/main/java/gov/hhs/cdc/trustedintermediary/etor/EtorDomainRegistration.java

```diff
@@ -103,11 +104,26 @@ DomainResponse handleDemographics(DomainRequest request) {
         return patientDemographicsController.constructResponse(patientDemographicsResponse);
     }
 
-    DomainResponse handleOrders(DomainRequest domainRequest) {
+    DomainResponse handleOrders(DomainRequest request) {
```

Comment:
Would it be worth extracting the common code in `handleOrders` and `handleDemographics` into a more generic `handleRequest`? To avoid code duplication and for reusability?

Reply:
Great idea. Let's do the refactoring once we have both controllers.